dockerfiles/createslim.sh

```diff
@@ -5,8 +5,8 @@ if [ ! -f "$LICENSE" ]; then
 fi
 
 docker rmi gravwell:slim #remove existing slim image
-docker pull gravwell/gravwell:latest # grabs latest gw image from dockerhub
-docker create --name slim gravwell/gravwell:latest #create temp container from latest image
+docker pull gravwell/gravwell:${VER} # grabs latest gw image from dockerhub
```

Comment:
Can we continue to use latest?
If not, we should probably change the comment here that says "latest"